### PR TITLE
Two patches from v1v

### PR DESCRIPTION
**Pull request by nre-ableton**
Incorporates changes from:

- https://github.com/AbletonDevTools/groovylint/pull/61
- https://github.com/AbletonDevTools/groovylint/pull/62

Our CI infrastructure doesn't yet support running on third-party forks, so I created this branch to be able to merge these patches to `master`.

@AbletonDevTools/gotham-city we've received a CLA from @v1v which I have forwarded to our legal team.

### CODE REVIEW

File: run_codenarc.py [hunk 1]
```diff
@@ -104,12 +104,16 @@ def _print_violations(package_file_path, violations):
     :return: Number of violations for the file.
     """
     for violation in violations:
+        try:
+            message = violation.find('Message').text
+        except AttributeError:
+            message = '[empty message]'
```
Comment (dgl-ableton) [inline run_codenarc.py]: Perhaps this should be changed to this? :) https://github.com/AbletonDevTools/groovylint/pull/62#discussion_r272606531

Comment (nre-ableton) [reply run_codenarc.py]: Sure, I guess. Fixed in 4e96dc2.

**Comment by dgl-ableton** [comment]
Could `41dfe32` go before `c07e582` so that the linting problems are fixed first and `c07e582` can run on its own? :)

**Comment by nre-ableton** [comment]
@dgl-ableton Sure, rebased + pushed.

**Comment by nre-ableton** [comment]
The reason for not squashing the commits is that it would effectively erase @v1v's work, and I wanted to keep that in the commit history. Though I guess we could squash, keep him as an author, and change the commit to make me the co-author.

**Pull request merged**